iodata/formats/qchemlog.py

```diff
@@ -56,7 +56,7 @@ def load_one(lit: LineIterator) -> dict:
     # build molecular orbitals
     # ------------------------
     # restricted case
-    if not data['unrestricted']:
+    if data['unrestricted'] is False:
```

Comment:
I believe this change goes against PEP8. See https://www.python.org/dev/peps/pep-0008/ (Look for `Don't compare boolean values to True or False`) Is there is a special reason for this change?

I'll take a closer look at the whole PR soon.

Reply:
Not any reason that I can recall now or that I have in my notes....probably I change it because it felt more "correct"  in my naive python knowledge, although according to PEP8 is the worst of possible options... I am going to change it as well as two more I've seen and commit the changes. Thanks for noticing!